### PR TITLE
[1.11] Configure Github CI to run tests on Windows

### PR DESCRIPTION
**Pull request by Sgitario**
Configured only for:
- PRs (similarly as done for Linux)
- Daily (only graalvm)

Not done for Ecosystem.

Backport of https://github.com/quarkus-qe/beefy-scenarios/pull/182

### CODE REVIEW

**Comment by Sgitario** [comment]
Daily run example in https://github.com/Sgitario/beefy-scenarios/actions/runs/819135711

File: .github/workflows/daily.yaml [hunk 1]
```diff
@@ -21,24 +22,65 @@ jobs:
...
+        with:
+          name: maven-repo
+          path: .
+      - name: Extract Maven Repo
+        shell: bash
+        run: tar -xzf maven-repo.tgz -C ~
       - name: Test in JVM mode
         run: |
-          mvn -V -B -s .github/mvn-settings.xml -fae clean verify
+          mvn -fae -V -B -s .github/mvn-settings.xml -fae clean verify
```
Comment (mjurc) [inline .github/workflows/daily.yaml:65]: Repeated `-fae`

File: .github/workflows/daily.yaml [hunk 2]
```diff
@@ -51,17 +93,130 @@ jobs:
...
+        with:
+          name: maven-repo
+          path: .
+      - name: Extract Maven Repo
+        shell: bash
+        run: tar -xzf maven-repo.tgz -C ~
       - name: Test in Native mode
         run: |
-          mvn -V -B -s .github/mvn-settings.xml -fae clean verify -Dnative \
+          mvn -fae -V -B -s .github/mvn-settings.xml -fae clean verify -Dnative \
```
Comment (mjurc) [inline .github/workflows/daily.yaml:106]: Again, duplication of `-fae`

**Review by mjurc — CHANGES_REQUESTED** [review]
Besides the two nitpicks, LGTM

File: .github/workflows/daily.yaml (hunk 1 above)
Comment (Sgitario) [reply .github/workflows/daily.yaml]: Fixed

File: .github/workflows/daily.yaml (hunk 2 above)
Comment (Sgitario) [reply .github/workflows/daily.yaml]: Fixed

**Comment by Sgitario** [comment]
@mjurc would you mind to take a look into this PR? Thanks in advance,

**Comment by mjurc** [comment]
Looks good to me, merging!

**Pull request merged**